atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/ConsecutiveNarrowTable.java

```diff
@@ -150,15 +150,15 @@ private static Cell cell(int index) {
 
     private static void storeDataInTable(ConsecutiveNarrowTable table, int numOverwrites) {
         int numRows = table.getNumRows();
-        IntStream.range(0, numOverwrites + 1).forEach($ -> {
-            for (int i = 0; i < numRows; i += PUT_BATCH_SIZE) {
-                final Map<Cell, byte[]> values =
-                        Tables.generateContinuousBatch(table.getRandom(), i, Math.min(PUT_BATCH_SIZE, numRows - i));
-                table.getTransactionManager().runTaskThrowOnConflict(txn -> {
+        table.getTransactionManager().runTaskThrowOnConflict(txn -> {
```

Comment:
Doesn't this mean we aren't actually generating dirty rows?  If we are running the entire thing in one transaction then all rows will have the same timestamp.

I think you could put the transaction boundary around the "for (int i" loop and probably accomplish what you want.